demos/atk-init.php

```diff
@@ -44,9 +44,9 @@
     $app->cdn['atk'] = $rootUrl . 'public';
 }
 
-// enable layout change.
-$layout = '\\atk4\\ui\Layout\\' . ($app->stickyGET('layout') ?: 'Maestro');
-$app->initLayout(new $layout());
+// allow custom layout override
+$layoutClass = $app->stickyGET('layout') ? 'atk4\ui\Layout\\' . $app->stickyGET('layout') : \atk4\ui\Layout\Maestro::class;
+$app->initLayout(\atk4\ui\Layout\Generic::factory([$layoutClass]));
```

Comment:
@ibelar is different class than a subtype of `\atk4\ui\Layout\Generic` passed in Travis tests?